optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/uni/UniConstraintStreamTest.java

```diff
@@ -454,7 +454,6 @@ public void groupBy_0Mapping1Collector_max() {
 
     @Test
     public void groupBy_1Mapping1Collector_count() {
```

Comment:
Interesting extra test coverage:
- groupKeyMapping returns primitive type: something like: groupBy(Person::getAge) and age is an int
- a groupKey that has no instances has no match: groupBy(Person::getCity) but here is no Person in New York.